assets/blocks/reader-registration/edit.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { TextControl, PanelBody } from '@wordpress/components';
+import { useBlockProps, InspectorControls } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import './editor.scss';
+
+export default function ReaderRegistrationEdit( {
+	setAttributes,
+	attributes: { placeholder, label },
+} ) {
+	const defaultPlaceholder = __( 'Enter your email address', 'newspack' );
+	const defaultLabel = __( 'Register', 'newspack' );
```

Comment:
There's no need to define `defaultPlaceholder` & `defaultLabel` – default values are already defined in `block.json`.

Reply:
You're right! Fixed in bce351eec2565c2624a380f7ae6abdae9347a17f